### PR TITLE
Feat(eos_cli_config_gen): Add BFD Slow-Timer Knob

### PR DESCRIPTION
**Pull request by ccsnw**
## Change Summary

Adding Slow-Timer knob to BFD config.
Fixing two minor typos in existing J2 template (2 space where only 1 space should be used)

## Related Issue(s)

Fixes #<N/A>

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Adding additional knob to configure slow-timer value in BFD

router_bfd:
<snip>
      slow_timer:
        description: Rate in milliseconds.
        type: int
        min: 2000
        max: 60000

## How to test
Molecule Test added

### User Checklist

- **Have checked eos cli order for command**
- Documentation **not** updated (minor knob)

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [o] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bfd.schema.yml [hunk 1]
```diff
@@ -81,3 +81,8 @@ keys:
               local_discriminator:
                 type: str
                 description: IPv4 address or 32 bit integer.
+      slow_timer:
+        description: Rate in milliseconds.
+        type: int
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bfd.schema.yml:86]:
```suggestion
        type: int
        convert_types:
          - str  
```

**Review by Shivani-gslab — DISMISSED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg [hunk 2]
```diff
@@ -3355,6 +3355,7 @@ peer-filter PF2
    30 match as-range 65000 result accept
 !
 router bfd
+   slow-timer 5000
    interval 900 min-rx 900 multiplier 50 default
    multihop interval 300 min-rx 300 multiplier 3
    local-address 192.168.255.1
```
Comment (Vibhu-gslab) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: According to EOS CLI `slow-timer` will come after `multihop interval`

```
   interval 900 min-rx 900 multiplier 50 default
   multihop interval 300 min-rx 300 multiplier 3
   slow-timer 5000
   local-address 192.168.255.1
```

You would have to change the order in eos/router-bfd.j2

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md [hunk 3]
```diff
@@ -5348,6 +5348,7 @@ policy-map type pbr PM_PBR_BREAKOUT
 ```eos
 !
 router bfd
+   slow-timer 5000
```
Comment (Vibhu-gslab) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md]: Please add `slow-timer` entry  in `/j2templates/documentation/router-bfd.j2`.

**Review by Vibhu-gslab — DISMISSED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg (hunk 2 above)
Comment (ccsnw) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: @Vibhu-gslab which EOS version are you testing the order on?

I used 4.26.3 and the order there is:

```
router bfd
   slow-timer 5000
   local-address 1.1.1.1
   session stats snapshot interval dangerous 1
   interval 300 min-rx 300 multiplier 10 default
   multihop interval 50 min-rx 50 multiplier 3
   !
   sbfd
      initiator interval 50 multiplier 5
!
```

Comment (Vibhu-gslab) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: @ccsnw I checked it on Software image version: 4.32.2F-38195967.4322F (engineering build)
@gmuloc

Comment (ccsnw) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: thanks, let me update this and also update my testswitch to 4.32

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg [hunk 4]
```diff
@@ -3355,6 +3355,7 @@ peer-filter PF2
    30 match as-range 65000 result accept
 !
 router bfd
+   slow-timer 5000
    interval 900 min-rx 900 multiplier 50 default
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: When trying to configure this the order is wrong. We are now attempting to match the order on EOS to make it easier to compare with a running config. Could you please move it?

```eos
(config-s-claus)#show sess diff
--- system:/running-config
+++ session:/claus-session-config
+router bfd
+   interval 900 min-rx 900 multiplier 50 default
+   multihop interval 300 min-rx 300 multiplier 3
+   slow-timer 5000
+   local-address 192.168.255.1
+   session stats snapshot interval 51
+   !
+   sbfd
+      local-interface Loopback0 ipv4 ipv6
+      initiator interval 500 multiplier 3
+      initiator measurement delay round-trip
+      reflector min-rx 600
+      reflector local-discriminator 155.1.3.1
smv033(config-s-claus)#
```

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg (hunk 4 above)
Comment (ccsnw) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: yup already WIP @ClausHolbechArista 

i was testing on 4.26.. where slow timer is above all the other commands

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md (hunk 3 above)
Comment (ccsnw) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md]: added bfd tuning section for it, like under router_bgp

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg (hunk 4 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: arg. if eos reorders between releases we will never hit the target :) Thanks.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by gmuloc — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg (hunk 4 above)
Comment (ccsnw) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg]: yup, would be great if the final eos output would follow the order of the suggestions, so in this case:

config-router-bfd)#?
  authentication  Configure BFD authentication
  hardware        Configure hardware-specific BFD parameters
  interval        Set transmit rate in milliseconds
  ip              IP config commands
  ipv6            IPv6 config commands
  local-address   Configure BFD local IP address
  multihop        Multihop BFD session
  peer            Specify peer of BFD session
  sbfd            Seamless BFD
  session         BFD session settings
  shutdown        Set BFD admin down globally
  slow-timer      Set slow timer rate in milliseconds
  vrf             Specify VRF encapsulating peers

>> auth before hardware before interval before ip.. etc :D

**Pull request merged**